### PR TITLE
Cnnetantwo 21 2

### PR DESCRIPTION
**Pull request by dsover**
enhance toggles to save to backend service

FIXES: CNNETANTWO-21 http://tickets.turner.com/browse/CNNETANTWO-21
**see #28 for testing information

screen shot 2017-08-11 at 3 26 07 pm

### CODE REVIEW

File: package.json [hunk 1]
```diff
@@ -1,6 +1,6 @@
 {
     "name": "cnn-antools-factors-fe",
-    "version": "0.1.0",
+    "version": "0.2.0",
```
Comment (jamsyoung) [inline package.json]: version shouldn't change here.  this is done when a release is created between the develop and master branches.

File: src/redux/actions/live_streams_actions.js [hunk 2]
```diff
@@ -100,3 +123,64 @@ export function cancelToggleOnLiveStream() {
...
+        const axiosClient = axios.create();
+        axiosClient.defaults.timeout = config.defaultAPITimeout_MS;
+
+        let brainApiUri = config.brainApi.uri,
+            postUrl = `${brainApiUri}/api/v1/livefeeds`,
+            payload = {
+                feedId: whichToggle,
+                isActive: valueToggledTo,
+                urls: [urls]
+            };
```
Comment (jamsyoung) [inline src/redux/actions/live_streams_actions.js]: Doesn't look like these change, can they be `const`s?

File: package.json (hunk 1 above)
Comment (dsover) [inline package.json]: This is changed in a merge to master I but the dev branch is locked...

Comment (jamsyoung) [inline package.json]: `master` and `develop` are now in sync.

File: src/redux/actions/live_streams_actions.js (hunk 2 above)
Comment (dsover) [inline src/redux/actions/live_streams_actions.js]: yup can do  and did :-)

**Pull request merged**